_data/_wmt_events.json

```diff
@@ -0,0 +1,161 @@
+[
+  {
+    "name": "Name of the event",
+    "id": "The slug for the event: Don't provide 'id' if a dedicated page doesn't need to be created for the event",
+    "description": "Description of the event",
+    "calls_for_papers_deadline": "Date format: yyyy-mm-dd, If the event has a calls For Papers",
+    "start_date": "Date format: yyyy-mm-dd",
+    "end_date": "Date format: yyyy-mm-dd",
+    "location": {
+      "type": "Type of the location: Offline, hybrid, online, etc.",
+      "name": "Country/City",
+      "event_attendance_mode": "OfflineEventAttendanceMode/OnlineEventAttendanceMode/Hybrid",
+      "url": "URL for the location where the event will happen"
+    },
+    "organizer": {
+      "type": "Association/Organization/etc.",
+      "name": "Name of the organizer",
+      "url": "URL about the organizer"
+    },
+    "future_tense_opening_paragraph": "An opening paragraph in future tense about the event, when/where it's happening.",
```

Comment:
Can we just autogenerate these in the .md?

For example:

```

dates = f'on { start_date }' if end_date == start_date else f'from { start_date } to { end_date }'

location = 'online' if ... else f'in { location_name }'

future_tense_opening_paragraph = f'{ name } will take place { dates} { location }.'

future_tense_opening_paragraph = f'{ name } took place { dates} { location }.'

```

And then `summaries` can be renamed `more_paragraphs`.